### ISSUE TITLE
Trademark Symbol

### ISSUE THREAD

**Issue by mindreeper2420** [almighty/almighty-ui#366]
The trademark symbol on the navigation item 'Hypothesis Engine' should use the HTML Name Code of `&trade;`. This will prevent any issues with the superscript positioning while using icons.

**Comment by joshuawilson** [comment]
@almighty/uxd can you please confirm with @qodfathr if we are going to keep the "TM" as I know the phrase existed as early as 2009.

**Comment by qodfathr** [comment]
It was just a joke. That tab isn't even part of the current UX plan. All of the names on the tabs, and the # of tabs, are just placeholders.

For now, we just need to work towards a set of "extensions" which are discovered at runtime and via their manifest/api/whatever provide a name to the system for display. Until we've got that, the "fake" placeholder tabs aren't too important (although having them be thematically correct helps).

In other words, don't stress out about this.

**Comment by sanbornsen** [comment]
Can we close this for now, in that case?

**Comment by qodfathr** [comment]
Fine with me to close.

**Comment by Mgranfie** [comment]
FYI... I will be updating the tab names this week.... and they will
probably continue to change... :)
